components/organisms/home/HomeTestimonials.vue

```diff
@@ -84,16 +85,18 @@
                   <NuxtLabel tag="span" class="font-bold text-base">{{ testimonial.author }}</NuxtLabel>
                   <NuxtLabel tag="span" class="text-sm dark:text-cloud-lighter">{{ testimonial.job }}</NuxtLabel>
                 </a>
-                <a :href="testimonial.jobUrl" target="_blank" rel="noopener sponsored" class="hidden xl:block">
+                <a :href="testimonial.jobUrl" :aria-label="`${testimonial.jobName} website`" target="_blank" rel="noopener sponsored" class="hidden xl:block">
```

Comment:
Is there a reason you prefer `aria-label` over `title` here?

Reply:
No reason. I can switch it to `title` instead. I didn't check to see if there was a convention for it in the project.